lib/monitor.js

```diff
@@ -134,8 +139,13 @@ internals.Monitor.prototype.start = function (callback) {
                 item.reporter = require(item.reporter);
             }
 
-            Hoek.assert(typeof item.reporter === 'function', 'reporter key must be a constructor function');
-            Hoek.assert(typeof item.events === 'object', 'reporter must specify events to filter on');
+            // attempt to upgrade hint to reporter name
+            var attributes = item.reporter.attributes || {};
```

Comment:
Sorry, when I said "do we need this" I mean the options into `Hoek.reach` use that instead as it's much easier, I just don't think we need any options in there.